data.json

```
@@ -2,6 +2,8 @@
   "Go": {
     "DefaultVersion": "go1.12.17",
     "VersionExpansion": {
+      "go1.16.0": "go1.16",
```

```suggestion
      "go1.16.0": "go1.16beta1",
```

When go1.16 comes out, this will be updated to go1.16 and stay there. The `go1.16` key on the next line will track the latest go1.16.x version.